reflex/utils/format.py

```diff
@@ -182,6 +183,24 @@ def format_string(string: str) -> str:
     return string
 
 
+def format_var(var: Var) -> str:
```

Comment:
Shouldn't this be the same as the Var.__str__ method? Should we move this code there? This was the intention of that method.

Reply:
hmm, it's a little different. i cribbed this from `Tag.format_prop`, so i assumed it must be doing something special. i'll see if i can combine them without breaking tests, so we can just use `str(var)` instead.